packages/integration-sdk-cli/src/utils/getSortedJupiterOneTypes.ts

```diff
@@ -141,3 +140,11 @@ export function collectGraphObjectMetadataFromSteps(
     mappedRelationships,
   };
 }
+
+function toRelationshipSetValue(r: StepRelationshipMetadata) {
```

Comment:
The main updates can be summarized in these two functions. Previously, `_type` was being used to de-dup doc rows.